src/screens/poa-form/styles.scss

```diff
@@ -47,9 +47,27 @@
   display: none !important;
 }
 
-.stepper div div div div { // HACK: Since the slider sets all styles inline.
-  a,
-  span {
-    line-height: 32px !important;
+.grommetux-box--direction-row.grommetux-box--responsive.button-box {
+  @media (max-width: 719px) {
+    flex-direction: column-reverse;
+    .button {
+      width: 100%;
+      max-width: 100%;
+      margin-bottom: 15px;
+    }
+  }
+}
+
+.stepper {
+  div div div div {
+    // HACK: Since the slider sets all styles inline.
+    a,
+    span {
+      line-height: 32px !important;
+    }
+  }
+
+  @media (max-width: 430px) {
```

Comment:
April requests that the "Back" button be hidden when the stepper _is_ visible (on larger screens).  I can file a separate issue for that though.